### PR TITLE
Feat/sqljs

### PR DESCRIPTION
**Pull request by delneg**
As discussed in https://github.com/delneg/fable-sqljs-absurdsql/issues/1

This sample provides code which should be as easy to run as 
`npm install && npm serve`

### CODE REVIEW

File: sqljs/src/App.fs [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+  | None ->
+    let! _ = api.runMany "INSERT INTO users (id,name, email) VALUES (?, ?, ?)" !!usersData
+    let! users = api.execRaw("SELECT * from users")
+    let mapped = users |> Seq.tryHead |> Option.map (UsersTable.mapUsers)
+    JS.console.log(mapped)
+  
+}
+init()
+|> Promise.catch (fun x -> JS.console.error(x))
+|> Promise.start
```
Comment (MangelMaxime) [inline sqljs/src/App.fs]: This section of the code should be formatted using 4 spaces indentations to be consistent with the others files.

**Review by MangelMaxime — COMMENTED** [review]
@alfonsogarciacaro 

Personally, I would prefer if this example is maintain by its creator because we are already having a lot of examples in this repos and we don't really maintain them...

Also, this example is using WASM + custom bindings + worker which is not something simple that I would expect in a minimal example.

In the past, we often decided to stop maintaining some of the examples by lack of human force and I fear that this will be the case again with all the examples that we keep adding in this repository.

Example:
- Pixi.js
- Electron
- etc.

**Comment by alfonsogarciacaro** [comment]
@MangelMaxime You're definitely right that we shouldn't overcrowd the examples as we don't maintain them very often, but @delneg has updated most of the examples to Fable 3 so he has some credentials to push a new one :)

In any case, another way to give visibility to fable-sqljs could also be a personal repository and a blog post that we promote through Fable's twitter account.

File: sqljs/src/App.fs (hunk 1 above)
Comment (delneg) [reply sqljs/src/App.fs]: Done

**Comment by delneg** [comment]
I've done formatting as requested, however I'm not sure how to proceed - I don't feel like writing a blog post atm, if this PR goes out of the scope of "samples" mindset then it can be ok to close it and/or leave the work in my own repo (https://github.com/delneg/fable-sqljs)

**Comment by alfonsogarciacaro** [comment]
Okay, let's merge it. Our future selves can always remove it if it becomes a maintenance burden :) Thank you @delneg!

**Pull request merged**